git/cmd.py

```diff
@@ -629,7 +636,13 @@ def execute(self, command,
 
         def _kill_process(pid):
             """ Callback method to kill a process. """
-            p = Popen(['ps', '--ppid', str(pid)], stdout=PIPE)
+            if sys.platform == 'win32':
+                CREATE_NO_WINDOW = 0x08000000
+                creationflags = CREATE_NO_WINDOW
+            else:
+                creationflags = None
+
+            p = Popen(['ps', '--ppid', str(pid)], stdout=PIPE, creationflags)
```

Comment:
I am afraid that using `creationflags` as positional argument doesn't actually end up in the correct spot. Also I thought that positional arguments after keyword args are not allowed.